SimPEG/EM/Analytics/FDEMDipolarfields.py

```diff
@@ -57,13 +71,27 @@ def E_from_ElectricDipoleWholeSpace(XYZ, srcLoc, sig, f, current=1., length=1.,
         return Ex, Ey, Ez
 
 
-def E_galvanic_from_ElectricDipoleWholeSpace(XYZ, srcLoc, sig, f, current=1., length=1., orientation='X', kappa=1., epsr=1.):
+def E_galvanic_from_ElectricDipoleWholeSpace(XYZ, srcLoc, sig, f, current=1., length=1., orientation='X', kappa=1., epsr=1., t=0.):
```

Comment:
for naming conventions, can we increase our assumptions at the end of the name (so when tab completing, you start with `ElectricDipoleWholeSpace` and can see the what things we can compute from it) 

e.g. `ElectricDipoleWholeSpace_E_galvanic`